examples/grpc-server-with-prometheus/client/client.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+
+	"github.com/grpc-ecosystem/go-grpc-prometheus"
+	pb "github.com/grpc-ecosystem/go-grpc-prometheus/examples/grpc-server-with-prometheus/protobuf"
+)
+
+func main() {
+	// Create a standard client metrics object.
+	grpcMetrics := grpc_prometheus.NewClientMetrics()
```

Comment:
are these metrics actually registered? I don't think so.

Reply:
```
var (
	// Create a metrics registry.
	reg = prometheus.NewRegistry()

	// Create some standard server metrics.
	grpcMetrics = grpc_prometheus.NewServerMetrics()

	// Create a customized counter metric.
	customizedCounterMetric = prometheus.NewCounter(prometheus.CounterOpts{
		Name: "demo_server_say_hello_method_handle_count",
		Help: "Total number of RPCs handled on the server.",
	})
)

func init() {
	// Register standard server metrics and customized metrics to registry.
	reg.MustRegister(grpcMetrics, customizedCounterMetric)
}
```
Is the above code can be guaranteed these metrics actually registered? I rewrite it.

Reply:
When i use default registry for prometheus, i put some codes in init function, such as:
```
var (
	customizedCounterMetric = prometheus.NewCounter(prometheus.CounterOpts{
		Name: "demo_server_say_hello_method_handle_count",
		Help: "Total number of RPCs handled on the server.",
	})
)
func init(){
prometheus.MusterRegister(customizedCounterMetric)
}
```
That can guarantee mertics actually registered.

Reply:
@Bplotka

Reply:
Sorry for delay. I don't get your question. All these things you wrote above are true. Just there is none 
```
reg.MustRegister(grpcMetrics) 
```
in this package

Reply:
@fengzixu still not addressed

Reply:
Oh, and even worse - we don't expose our metric endpoint here (like we do on server) so these metrics are useless. Let's either drop them or serve them from here as well.

These metrics are for client monitoring.